frontend/src/global_styles/content/_modal.lsg

```diff
@@ -81,29 +77,32 @@
       </div>
     </fieldset>
   </div>
-  <div class="op-modal--footer">
-    <button name="button" type="submit" class="button -highlight">Save</button>
+  <div class="spot-modal--footer spot-action-bar">
+    <div class="spot-action-bar--right">
+      <button name="button" type="submit" class="button button_no-margin -highlight spot-action-bar--action">Save</button>
+    </div>
   </div>
 </div>
 ```
 
 ## Modals: Danger zone highlighting
```

Comment:
Thanks for also updating the LSG 👍  This whole block can be removed, as there is no such thing as a danger modal any more.

Reply:
removed the highlighted one, but kept the `danger zone` to indicated the button style matter

also added a close footer to the standard one.